envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -244,10 +248,10 @@ abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
 
         fun callService(
```

Comment:
Have you thought about inlining those methods and leaving only `Ingress/EgressOperations`?

Reply:
Yeah, I did. But such a change would require changes in most of our tests that are based on the old setup. I decided it's better to focus on migrating tests to the new setup than to apply temporary micro cleanups ;)

Reply:
To be clear: `EnvoyControlTestConfiguration` and `BaseEnvoyTest` will be removed once we migrate everything to the new setup.